.ci/docs

```diff
@@ -27,8 +27,8 @@ pipeline {
                 sh '''
                 eval "$(pyenv init -)"
                 pyenv --version
-                pyenv install --skip-existing 2.7.15
-                pyenv shell 2.7.15
+                pyenv install --skip-existing 3.6.8
```

Comment:
Is there a way to pin to latest 3.6 ?  would that be better or worse than pinning to a specific version?

Reply:
latest would be a moving target(ok, it wouldn't move that much).
I guess a static, that we can update when needed is better.